backend/entity_routes.py

```
@@ -185,137 +263,251 @@ async def fuzzy_search_by_name(
+    # ── Recent alerts referencing this entity ─────────────────────────────────
+    alert_rows = []
+    if settings.ALERT_SYSTEM_ENABLED:
+        try:
+            alert_rows = (
+                db.query(Alert.id, Alert.anomaly_type, Alert.created_at)
+                .filter(Alert.affected_entities.contains([entity_id]))
+                .filter(Alert.created_at >= since_dt)
+                .order_by(Alert.created_at.asc())
+                .all()
+            )
+        except Exception:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)